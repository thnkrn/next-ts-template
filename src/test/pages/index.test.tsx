import IndexPage from 'pages/index';
import { render } from 'test/util';

describe('Index', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(<IndexPage />);

    expect(asFragment()).toMatchSnapshot();
  });
});
