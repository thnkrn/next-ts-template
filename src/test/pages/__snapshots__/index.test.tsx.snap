// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should match snapshot 1`] = `
<DocumentFragment>
  <div>
    <style
      data-cmpld="true"
    >
      ._otyrgktf{margin-bottom:20px}
    </style>
    <header
      class="_otyrgktf"
    >
      <nav>
        <a
          href="/"
        >
          Home
        </a>
         | 
        <a
          href="/about"
        >
          About
        </a>
         | 
      </nav>
    </header>
    <style
      data-cmpld="true"
    >
      ._k48p8n31{font-weight:bold}
    </style>
    <h1
      class="_k48p8n31"
    >
      Hello Next.js ðŸ‘‹
    </h1>
    <p>
      <a
        href="/about"
      >
        About
      </a>
    </p>
    <style
      data-cmpld="true"
    >
      ._19pkgktf{margin-top:20px}
    </style>
    <footer
      class="_19pkgktf"
    >
      <hr />
      <span>
        Footer Here
      </span>
    </footer>
  </div>
</DocumentFragment>
`;
